<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="22.691" tests="2" failures="2">
  <testsuite name="Root Suite" timestamp="2020-03-20T21:25:57" tests="0" file="cypress/integration/apps/lims/LM02.js" failures="0" time="0">
  </testsuite>
  <testsuite name="2 - I should be able to create a new worksheet" timestamp="2020-03-20T21:25:57" tests="2" failures="2" time="22.691">
    <testcase name="2 - I should be able to create a new worksheet LM-2-1 Should be able to create a new worksheet in list view" time="22.691" classname="LM-2-1 Should be able to create a new worksheet in list view">
    </testcase>
    <testcase name="2 - I should be able to create a new worksheet &quot;before each&quot; hook for &quot;LM-2-2 Should be able to create a new worksheet in card view&quot;" time="0" classname="&quot;before each&quot; hook for &quot;LM-2-2 Should be able to create a new worksheet in card view&quot;">
      <failure message="Timed out after waiting &apos;20000ms&apos; for your remote page to load.

Your page did not fire its &apos;load&apos; event within &apos;20000ms&apos;.

You can try increasing the &apos;pageLoadTimeout&apos; value in &apos;cypress.json&apos; to wait longer.

Browsers will not fire the &apos;load&apos; event until all stylesheets and scripts are done downloading.

When this &apos;load&apos; event occurs, Cypress will continue running commands.

Because this error occurred during a &apos;before each&apos; hook we are skipping the remaining tests in the current suite: &apos;2 - I should be able to cre...&apos;" type="CypressError"><![CDATA[CypressError: Timed out after waiting '20000ms' for your remote page to load.

Your page did not fire its 'load' event within '20000ms'.

You can try increasing the 'pageLoadTimeout' value in 'cypress.json' to wait longer.

Browsers will not fire the 'load' event until all stylesheets and scripts are done downloading.

When this 'load' event occurs, Cypress will continue running commands.

Because this error occurred during a 'before each' hook we are skipping the remaining tests in the current suite: '2 - I should be able to cre...'
    at cypressErr (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:138017:3)
    at timedOutWaitingForPageLoad (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:126147:20)
    at https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:126943:16
    at tryCatcher (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:164895:23)
    at https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:160036:41
    at tryCatcher (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:163008:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
    <testcase name="2 - I should be able to create a new worksheet &quot;after all&quot; hook for &quot;LM-2-2 Should be able to create a new worksheet in card view&quot;" time="0" classname="&quot;after all&quot; hook for &quot;LM-2-2 Should be able to create a new worksheet in card view&quot;">
      <failure message="cy.exec(&apos;esp -s -P 443 -H chris.test.l7informatics.com -u admin@localhost -p L4b7R0cks! seed -f -d cypress/fixtures/apps/lims/LM02teardown.yml&apos;) failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
Traceback (most recent call last):
  File &quot;/Users/cj/bitrepos/esp/develop/bin/.venv/bin/esp&quot;, line 11, in &lt;module&gt;
    load_entry_point(&apos;espclient&apos;, &apos;console_scripts&apos;, &apos;esp&apos;)()
  File &quot;/Users/cj/bi...

Because this error occurred during a &apos;after all&apos; hook we are skipping the remaining tests in the current suite: &apos;2 - I should be able to cre...&apos;" type="CypressError"><![CDATA[CypressError: cy.exec('esp -s -P 443 -H chris.test.l7informatics.com -u admin@localhost -p L4b7R0cks! seed -f -d cypress/fixtures/apps/lims/LM02teardown.yml') failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
Traceback (most recent call last):
  File "/Users/cj/bitrepos/esp/develop/bin/.venv/bin/esp", line 11, in <module>
    load_entry_point('espclient', 'console_scripts', 'esp')()
  File "/Users/cj/bi...

Because this error occurred during a 'after all' hook we are skipping the remaining tests in the current suite: '2 - I should be able to cre...'
    at cypressErr (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:138017:3)
    at https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:125593:26
    at tryCatcher (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:163012:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (https://chris.test.l7informatics.com/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
  </testsuite>
</testsuites>